config/locales/en.yml

```diff
@@ -492,6 +492,58 @@ en:
         serial_orders:
           false: Episodic
           true: Serial
+        subscribe_link:
+          apple: Apple Podcasts
+          spotify: Spotify
+          overcast: Overcast
+          pocketcasts: Pocket Casts
+          youtube: Youtube Music
+          youtube_feed: Youtube Music (feed)
```

Comment:
is there a different way to label this to make it known to the user that this option uses a `feedURL` instead of a `uniquePlatformID`? only other platforms that do this below are Podcast Guru and PodLP.